WcaOnRails/app/views/competitions_mailer/notify_wcat_of_confirmed_competition.html.erb

```diff
@@ -4,6 +4,13 @@
   <h2 class="alert">This competition is marked as <%= @competition.championships.sort.map(&:name).to_sentence %></h2>
   <% end %>
 
+<% @competition.delegates.map do |delegate| %> 
+  <% if Team.probation.current_members.map { |member| member.user_id }.include?(delegate.id) %>
+  <p>  </p>
```

Comment:
The <p> is left over. I'll remove this when modifying the email with the other items.